### PR TITLE
[CAGE-4812] Small UI Changes to Carto Map

### PR DESCRIPTION
**Pull request by ccson**
https://thinkneardev.atlassian.net/browse/CAGE-4812

### CODE REVIEW

**Comment by ccson** [comment]
Jinja2 was rendering the macros, and not Mustache. 

https://shekhargulati.com/2014/05/07/using-python-flask-jinja2-with-mustache/

File: templates/index.html [hunk 1]
```diff
@@ -180,10 +181,10 @@
                               var testTooltip = layer.leafletMap.viz.addOverlay({
                                   type: 'tooltip',
                                   layer: sublayer,
-                                  template: '<div class="cartodb-tooltip-content-wrapper"><h4>{{brand_label}}</h4>' +
-                                            '<h3>RADII</h3><h4>{{width_category}}</h4></div>',
+                                  template: '{% raw %}<div class="cartodb-tooltip-content-wrapper"><h4>{{brand_label}}</h4>' +
```
Comment (d3cay1) [inline templates/index.html]: Why are these raw template tags here?

Comment (ccson) [reply templates/index.html]: @d3cay1 

https://shekhargulati.com/2014/05/07/using-python-flask-jinja2-with-mustache/

Comment (ccson) [reply templates/index.html]: These popups weren't rendering the width_category and brand_label as well.

File: templates/index.html [hunk 2]
```diff
@@ -624,24 +633,19 @@
...
-        dataType: JSON
+    $('.ui.filter.text.input.zip-form').form({
+        inline: true,
+        fields: {
+            zip: {
+                identifier: 'zip',
+                rules: [
+                    {
+                        type: 'regExp[/(^$|^(\\d{5})(,\\d{5})*)$/]',
+                        prompt: 'Field must be 5-digit zip codes delimited by commas'
```
Comment (d3cay1) [inline templates/index.html]: 👍

File: templates/index.html [hunk 3]
```diff
@@ -52,8 +52,9 @@
         #visualize-button:active { background-color: #EC9124; }
 
         /* For Filter Select Dropdowns */
-        .ui.multiple.dropdown {
```
Comment (d3cay1) [inline templates/index.html]: I think you can get rid of `multiple dropdown` as a class everywhere now

File: templates/index.html [hunk 4]
```diff
@@ -391,19 +395,19 @@
...
-    <select id="brand" class="ui multiple search dropdown brand" multiple="">
+    <select id="brand" class="ui filter multiple search dropdown brand" multiple="">
     </select>
 
     <br><br>
 
     <p>Radius</p>
 
-    <select id="distance" class="ui multiple search dropdown distance" multiple="">
+    <select id="distance" class="ui filter multiple search dropdown distance" multiple="">
```
Comment (d3cay1) [inline templates/index.html]: Example: you should be able to remove `multiple` and `dropdown` here.

**Pull request merged**